packages/core/src/command/patient-import/patient-import.ts

```diff
@@ -1,16 +1,11 @@
 import { PatientDemoData } from "../../domain/patient";
+import { PatientImportStatus } from "../../domain/patient/patient-import";
 
-export type JobStatus = "waiting" | "processing" | "completed" | "failed";
-
-export type PhaseStatus = "processing" | "completed" | "failed";
+export type PatientRecordStatus = "processing" | "successful" | "failed";
 
 export type PatientRecord = {
   patientId: string;
-  patientQueryStatus?: PhaseStatus;
-  documentQueryStatus?: PhaseStatus;
-  documentRetrevialStatus?: PhaseStatus;
-  documentConversionStatus?: PhaseStatus;
-  cleanUpStatus?: PhaseStatus;
```

Comment:
Not being used, so reckoned we should leave only what we're using for now. Can adjust as we finish implementing the flow.